README.md

```diff
@@ -43,15 +43,16 @@ https://nodejs.org/calendar
 <sup>Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.</sup>
 
 The calendar is maintained by:
-- Bryan Hughes [@nebrius](https://github.com/nebrius)
+<!-- sorted by GitHub handle -->
```

Comment:
Optional, but if you put the Github ID first, as in the [nodejs/node README](https://github.com/nodejs/node/#tsc-technical-steering-committee), the ordering should be more clear.

Reply:
Cool. I was leaning that way in my head but didn't know of any other list displayed like that. Updated now.